packages/wallet-standard/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@glow-xyz/wallet-standard",
-  "version": "0.0.3",
+  "version": "0.1.0",
```

Comment:
Bumped this to 0.1.0 since there is an internal breaking change --

You just do something like this in Glow now:
```ts
import { initialize } from '@glow-xyz/wallet-standard';

const glow = new GlowAdapter();
window.glow = glow;
initialize(glow);
```